cmd/runhcs/shim.go

```diff
@@ -321,3 +326,12 @@ var shimCommand = cli.Command{
 		return cli.NewExitError("", code)
 	},
 }
+
+// escapeArgs makes a Windows-style escaped command line from a set of arguments
+func escapeArgs(args []string) string {
+	escapedArgs := make([]string, len(args))
```

Comment:
You could use a `strings.Builder` and avoid the additional array alloc

Reply:
@jterry75 Happy to change if you think it's important enough.